Data-Science/Ray/Ray-CPU/Fibonacci/README.md

```diff
@@ -0,0 +1,44 @@
+## Ray Fibonacci Example
+
+##### This example demonstrates the comparison between executing tasks locally and distributedly using Ray.
+
+### Prerequisites:
+* Ensure you are using a Data Science notebook environment in Kubeflow. 
+* Ray client and server versions must match. Typically, `ray --version` can be used to verify the installed version. 
+* Activate the Ray-specific Python kernel in your notebook environment. 
+* For optimal performance, ensure the working directory contains only the necessary files for the Ray job.
```

Comment:
I think for bullet point we need to specify which exact directory and what necessary files are required in that directory for the Ray job.

Reply:
Actually there is no exact directory, I was trying to say that it is better to use a dedicated directory containing only the necessary files for each job, because each example pushes its current dir to Ray cluster as in below:
```
    job_id = c.submit_job(
        entrypoint=f"python ray-fibonacci-example.py {n}",
        runtime_env={
            "working_dir": "./",     <==== This transfers all files in the current directory to the Ray cluster.
        },
        entrypoint_num_cpus=3
    )
```

I notice that some people submit jobs from common/root directory that leads pushing too many unrelated files to Ray cluster. This causes object-memory issues.

Maybe my wording did not reflect this properly, can you help with better explanation?

Reply:
Thanks, Sercan! Will it be technically accurate to suggest users to run the tutorials from the /<username> directory? 

Maybe something like this, "To ensure optimal performance, use dedicated directories containing only the essential files needed for that job submission as a working directory. For example, If you do not see the Fibonacci folder in the <username> directory, copy the folder from the /shared/ezua-tutorials/Data-Science/Ray directory into the <username> directory. The /shared directory is accessible to all users. Editing or running examples from the /shared directory is not advised. The <username> directory is specific to you and cannot be accessed by other users."

I think it is okay if we want to keep instructions short here and directly add these detailed explanation in the documentation site for Ray tutorials too.

Reply:
Thank you for correcting this very detailed. I have used the short version `To ensure optimal performance, use dedicated directories containing only the essential files needed for that job submission as a working directory` in readme files. As you mentioned, we can use detailed version in our documentation site.

Is this good for you?